presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java

```diff
@@ -265,6 +265,7 @@ public static Block createBooleansBlock(Iterable<Boolean> values)
         return builder.build();
     }
 
+    // It is recommended to have positionCount >= 10 when nullRate > 0, so that the nullRate can be close to the desired value.
```

Comment:
@yingsu00 

nit: consider shortening to `Use positionCount >= 10 when nullRate > 0...`

What does `nullRate can be close to the desired value` mean?

Reply:
@mbasmanova thanks for the suggestion. By `nullRate can be close to the desired value`, I meant that the actual null rate would be more accurate(close to the specified nullRate) when positionCount is larger.

Reply:
@yingsu00 Maybe write it like `Make sure positionCount is sufficiently large if nullRate is small`.

Reply:
Thanks @mbasmanova. I updated the comment to be 
```
// Note: Make sure positionCount is sufficiently large if nullRate is greater than 0
```